client/src/components/organisms/AdminRequestGroupBrowser.tsx

```diff
@@ -1,8 +1,9 @@
 import { Dropdown, Spinner } from "react-bootstrap";
-import { gql, useQuery } from "@apollo/client";
+import { gql, useMutation, useQuery } from "@apollo/client";
```

Comment:
Not sure if this is just one of those things you always gotta import, (I'm not very familiar with GraphQL) but I don't think useQuery is explicitly used anywhere 👀